sound/soc/sof/debug-dsp-ops.c

```diff
@@ -8,6 +8,33 @@
 #include "sof-priv.h"
 #include "ops.h"
 
+static int sof_dsp_ops_boot_firmware(struct snd_sof_dev *sdev)
+{
+	const char *fw_filename;
+	int ret;
+
+	sdev->first_boot = true;
+
+	if (!sdev->test_profile.fw_path || !sdev->test_profile.fw_name) {
+		dev_dbg(sdev->dev, "Invalid firmware path or filename\n");
```

Comment:
well, it is not invalid, but either of them is missing. But which one? This would be asked by the user (who might not even see the error since it is in debug level...)